timelock-agent/src/main/java/com/palantir/atlasdb/timelock/paxos/TimeLockPaxosExecutors.java

```diff
@@ -27,7 +27,7 @@
 
 final class TimeLockPaxosExecutors {
     @VisibleForTesting
-    static final int MAXIMUM_POOL_SIZE = 100;
+    static final int MAXIMUM_POOL_SIZE = 384;
```

Comment:
Do we want a comment for this constant so we can understand how we got here in the future?

Reply:
Yep. This was basically derived from looking at the spikes on internal metrics platform, and was selected to permit most instances where a spike in shared executor tasks was serviced and the system recovered afterward (the idea being that the pool being full effectively acts as a QoS mechanism against the bad node).

Reply:
I'll add a comment.